test/src/graph/nodes/TransformPointsNodeTest.cpp

```diff
@@ -0,0 +1,89 @@
+#include <RGLFields.hpp>
+#include <graph/Node.hpp>
+#include <gtest/gtest.h>
+#include <repr.hpp>
+#include <utils.hpp>
+
+class TransformPointsNodeTest : public RGLAutoCleanupTestWithParam<std::tuple<int, rgl_mat3x4f>>, public RGLPointTestHelper {
+};
+
+INSTANTIATE_TEST_SUITE_P(
+    TransformPointsNodeTests, TransformPointsNodeTest,
+    testing::Combine(
+        testing::Values(1, 10, 100000),
+        testing::Values(identityTestTransform, translationTestTransform, rotationTestTransform, scalingTestTransform, complexTestTransform)));
```

Comment:
300 ms total. How about `1, 100, 10000`?

Reply:
I am fine with everything greater than the number of CU in a typical user GPU.

Reply:
Good point. Let's define this value. At the moment, this would be 20k.

Reply:
Let's define it as 20k for now in some test utils const.